front/lib/actions/mcp_internal_actions/constants.ts

```diff
@@ -66,6 +67,11 @@ export const INTERNAL_MCP_SERVERS: Record<
     isDefault: true,
     flag: "mcp_actions",
   },
+  hubspot: {
+    id: 6,
```

Comment:
looks like "think" is already id 6, I guess we need a unique one no ?

Reply:
Indeed 🙏🏻 
Will rebase!